backend/src/main/java/com/ambacam/service/TypeRequeteService.java

```diff
@@ -0,0 +1,116 @@
+package com.ambacam.service;
+
+import com.ambacam.exception.ResourceBadRequestException;
+import com.ambacam.exception.ResourceNotFoundException;
+import com.ambacam.model.TypeRequete;
+import com.ambacam.repository.TypeRequeteRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional(rollbackFor = Exception.class)
+public class TypeRequeteService {
+
+
+	@Autowired
+	private TypeRequeteRepository typeRequeteRepository;
+
+	/**
+	 * Create a typeRequete
+	 * 
+	 * @param typeRequete
```

Comment:
Add a description of the parameter typeRequete.
e.g: `@param typeRequete The object we want to create`;
We had a similar discussion on a previous pull request.
You can check this lin [http://www.oracle.com/technetwork/articles/java/index-137868.html](Doc)
In order, if you used Intellij, it will notice you about that kind of warning.